### DISCUSSION TITLE
Why aren’t links styled differently?

### DISCUSSION THREAD

**Discussion by xplosionmind** [omnivore-app/omnivore#1392] [Q&A]
By reading an article inside Omnivore, links are disguised within the text since they do not have any different graphical styling.

The only way to find a link is to pass the cursor over all of the text until the pointer changes into indicating that the element under it is clickable.

While the reading experience can become less distracting, I believe users should be given the choice whether to uniform links styling with the text or to make them pop out: #1391.

**Comment by jacksonh** [comment]
Hey, do you mind posting a screenshot? I wonder if some of our CSS got messed up. 

There's also a cool idea to add an icon beside links that would let you click to add to the library.

**Reply by xplosionmind** [reply]
In [this article](https://www.theatlantic.com/technology/archive/2022/02/future-internet-blockchain-investment-banking/621480) there is a link in “imagine a COVID-19 memorial”, and this is what I see (nothing changes even when hovering):

![Screenshot 2022-11-03 at 3 42 12 PM](https://user-images.githubusercontent.com/42862428/199752129-d55d3fb9-d397-43a0-bb58-6c5a15d42092.jpg)
![Screenshot 2022-11-03 at 3 41 59 PM](https://user-images.githubusercontent.com/42862428/199752140-fec82517-03b6-4ff1-96ae-5e9f2f7652f6.jpg)

**Comment by jacksonh** [comment]
Hey @xplosionmind  are you still seeing this? I wonder if with that article its the paywall blocker thing injecting some CSS. This is how that article looks for me. We did have some CSS bugs last week that we reverted.


<img width="908" alt="Screen Shot 2022-11-07 at 15 15 36" src="https://user-images.githubusercontent.com/75189/200247872-eadf390b-e82f-4747-b4fa-af5afdd04fae.png">

**Reply by xplosionmind** [reply]
I am having the same problem with all of the articles I saved in my Omnivore account.

**Comment by jacksonh** [comment]
@xplosionmind thanks, and you are on Firefox, right? Windows or MacOS?

**Reply by xplosionmind** [reply]
Lates Firefox on latest MacOS

**Reply by Bad3r** [reply]
I can't reproduce this issue using Firefox 107 on Linux. @xplosionmind can you please copy the article URL and add it directly into Omnivore from the web app and not from the browser plugin. Do you have the same issue? If yes, please provide the link to the article used.
